Tests/test.js

```diff
@@ -0,0 +1,29 @@
+const { batteryStatus } = require("../bms-monitor");
+const {limits}=require('../inputParameterLimits');
+const {outPutLogs}=require('../outputLogs');
+const assert =require('assert');
+
+const testBatteryCondition=batteryStatus(limits,outPutLogs);
+
```

Comment:
The conditions tested in this file look comprehensive :)
Try doing them in mocha

Reply:
Ok sir will do it